lib/tests/test_transaction.py

```diff
@@ -130,6 +130,42 @@ def test_tx_signed(self):
 
         self.assertEqual(tx.estimated_size(), 191)
 
+    def test_tx_nonminimal_scriptSig(self):
+        # The nonminimal push is the '4c41...' (PUSHDATA1 length=0x41 [...]) at
+        # the start of the scriptSig. Minimal is '41...' (PUSH0x41 [...]).
+        expected = {
+            'inputs': [{'address': Address.from_pubkey('034a29987f30ad5d23d79ed5215e034c51f6825bdb2aa595c2bdeb37902960b3d1'),
+                        'num_sig': 1,
+                        'prevout_hash': 'a3c30d58e88edb283c702a5ae488acd2f3b72219253aaf39318183bd6083b842',
+                        'prevout_n': 0,
+                        'pubkeys': ['034a29987f30ad5d23d79ed5215e034c51f6825bdb2aa595c2bdeb37902960b3d1'],
+                        'scriptSig': '4c41151dc44bece88c5933d737176499209a0b1688d5eb51eb6f1fd9fcf2fb32d138c94b96a4311673b75a31c054210b2058735ce6c12e529ddea4a6b91e4a3786d94121034a29987f30ad5d23d79ed5215e034c51f6825bdb2aa595c2bdeb37902960b3d1',
+                        'sequence': 4294967294,
+                        'signatures': ['151dc44bece88c5933d737176499209a0b1688d5eb51eb6f1fd9fcf2fb32d138c94b96a4311673b75a31c054210b2058735ce6c12e529ddea4a6b91e4a3786d941'],
+                        'type': 'p2pkh',
+                        'x_pubkeys': ['034a29987f30ad5d23d79ed5215e034c51f6825bdb2aa595c2bdeb37902960b3d1']}],
+            'lockTime': 1331787,
+            'outputs': [{'address': Address.from_pubkey('034a29987f30ad5d23d79ed5215e034c51f6825bdb2aa595c2bdeb37902960b3d1'),
```

Comment:
nb. I used from_pubkey instead of from_string since this was testnet, so I only had access to the `bchtest:` address.

Reply:
Ha.. it works. :)